src/Ims/README.md

```diff
@@ -0,0 +1,164 @@
+You have to be [logged in](../..) and have relevant permissions.
+
+### DIRECT MESSAGE LISTING
+
+```php
+$listing = \ATDev\RocketChat\Ims\Im::listing();
+
+if (!$listing) {
+
+	// Log the error
+	$error = \ATDev\RocketChat\Ims\Im::getError();
+}
+```
+
+### CREATE DIRECT MESSAGE SESSION
+
+```php
+$im = new \ATDev\RocketChat\Ims\Im();
+$im->setUsername("[USERNAME]");
+$im->setUsernames("username_first, username_second, username_third");
+
+$result = $im->create();
+
+if (!$result) {
+
+	// Log the error
+	$error = $im->getError();
+}
+```
+
+### OPEN DIRECT MESSAGE
+
+```php
+$im = new \ATDev\RocketChat\Ims\Im("[DIRECT MESSAGE ID]");
+
+$result = $im->open();
+
+if (!$result) {
+
+	// Log the error
+	$error = $im->getError();
+}
+```
+
+### CLOSE DIRECT MESSAGE
+
+```php
+$im = new \ATDev\RocketChat\Ims\Im("[DIRECT MESSAGE ID]");
+
+$result = $im->close();
+
+if (!$result) {
+
+	// Log the error
+	$error = $im->getError();
+}
+```
+
+### GET COUNTERS OF DIRECT MESSAGES
+
+```php
+$im = new \ATDev\RocketChat\Ims\Im("[DIRECT MESSAGE ID]");
+$im->setUsername("[USERNAME]");
+
+$result = $im->counters();
+
+if (!$result) {
+
+	// Log the error
+	$error = $im->getError();
+}
+```
+
+### GET MESSAGES FROM A DIRECT MESSAGE
+
+```php
+$im = new \ATDev\RocketChat\Ims\Im("[DIRECT MESSAGE ID]");
+$im->setLatest("2016-09-30T13:42:25.304Z");
```

Comment:
I don't think these are the properties for the `Im` object. These are filters, so we should pass them to `history` method. I think we can specify the filter as array.